content/source/docs/enterprise/saml/configuration.html.md

```diff
@@ -13,6 +13,10 @@ SAML requires the configuration of two parties:
 
 Terraform Enterprise is configured as the Service Provider.
 
+
+
+~> For MS ADFS specific instructions follow [this link](./adfs.html)
```

Comment:
Instead of linking directly to ADFS here, I'm thinking we should say "For instructions for specific IdPs, see [Identity Provider Configuration](https://www.terraform.io/docs/enterprise/saml/identity-provider-configuration.html)."

Thoughts?

Reply:
Yeah I Was about to suggest that we remove these lines entirely, but I think your idea is better!

Reply:
Awesome.